sdk-docs/getting-started/creating-a-new-extension.md

```diff
@@ -33,6 +33,8 @@ nx g @trackunit/iris-app:extend [name-of-your-extension]
   - **Report** Allows you to add a new report to the Reports screen in Trackunit Manager.
   - **Admin** Allows you to add a new tab in the adminstration page in Trackunit Manager.
 
+    ![](../../which%20iris%20app%20extension%20should%20be%20generated_.png)
```

Comment:
You need to include images like this: https://github.com/Trackunit/developer-hub/blob/e11fd14254aa0811fcdee7badb463b83e618a03a/sdk-docs/getting-started/extension-points.md#L26

The readme cli doesn't upload images :-(

Reply:
Done